modules/nw-osp-loadbalancer-source-ranges.adoc

```diff
@@ -0,0 +1,8 @@
+// Module included in the following assemblies:
+// * networking/nw-osp-loadbalancer-limitations.adoc
+
+:_content-type: CONCEPT
+[id="nw-osp-loadbalancer-source-ranges_{context}"]
+= `loadBalancerSourceRanges`
+
+`.spec.loadBalancerSourceRanges` can be used to restrict traffic allowed to be passing through the load balancer based on source IP. This property will only work with Amphora Octavia provider. In case of OVN Octavia provider the option will be ignored and traffic will be unrestricted.
```

Comment:
We try to avoid monospace markup in headings due to portal rendering issues. :\ 
```suggestion
[id="nw-osp-loadbalancer-source-ranges_{context}"]
= Load balancer source ranges

Use the `.spec.loadBalancerSourceRanges` property to restrict the traffic that can pass through the load balancer according to source IP. This property is supported for use with the Amphora Octavia provider only. If your cluster uses the OVN Octavia provider, the option is ignored and traffic is unrestricted.
```